### PR TITLE
Allows the Coroner's Ritual Knife to go in the Medical belt!

### PR DESCRIPTION
**Pull request by DrAmazing343**
## About The Pull Request
Allows the Ritual Knife from the Coroner's Mortidrobe contraband section to be stored within Medical Belts like any other surgical tool.
## Why It's Good For The Game
Coroners using the Ritual Knife adds excellent flavor to the job. Allowing it to sit upon their Medical Belt in place of, say, a scalpel grants convenience to those wishing to roleplay out a more wicked or occultic Coroner without having to place the Ritual Knife in their bag, leaving an odd spot open on their Medical Belt.
## Changelog
:cl: Jane
qol: The Coroner's Ritual Knife can now sit upon Med Belts!
/:cl:

### CODE REVIEW

**Comment by EricZilla** [comment]
Since you are in there already... do you think you could let coroner labcoat hold shovels too? Basically the same reason, it's the only morbid bone saw and it's a mess to lug around

**Comment by DrAmazing343** [comment]
> Since you are in there already... do you think you could let coroner labcoat hold shovels too? Basically the same reason, it's the only morbid bone saw and it's a mess to lug around

Changed appropriately, and fucked with the title a bit. Hopefully this clears all checks and you'll have your shovels slung over your shoulder in no time!

File: code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -286,6 +286,7 @@
 		/obj/item/surgicaldrill,
 		/obj/item/tank/internals/emergency_oxygen,
 		/obj/item/wrench/medical,
+		/obj/item/knife/ritual
```
Comment (00-Steven) [inline code/game/objects/items/storage/belt.dm:289]: missing trailing comma
```suggestion
		/obj/item/knife/ritual,
```

**Review by 00-Steven — CHANGES_REQUESTED** [review]

**Comment by DrAmazing343** [comment]
Okay, so, good news bad news; The bone shovel overlay is mirrored on the east and west sided sprites because of how the shovel displays on the back when worn on the belt. I think I can fix this by un-mirroring it, and then mirroring the back sprites on north and south to make having both shovels equipped at the same time crossing to make an X shape. This will be a separate PR and this PR is going to be back to just the ritual knife, but the other should be out soon!

**Comment by DrAmazing343** [comment]
alright we should be back to just our initial thing after like 6 commits, excellent job party people now lemme branch another

**Pull request merged**